src/tests/poc/include/standard/PortConnexion.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "vector"
+#include "string"
+
+using namespace std;
+
+class ComponentFiller;
+class PortConnexion
+{
+private:
+    pair<ComponentFiller*, string> componentAndPort1; // TODO : à terme, ne pas référencer directement des fillers
+    pair<ComponentFiller*, string> componentAndPort2;
+public:
+    PortConnexion(pair<ComponentFiller*, string> componentAndPort1, pair<ComponentFiller*, string> componentAndPort2)
+            : componentAndPort1(std::move(componentAndPort1)), componentAndPort2(std::move(componentAndPort2))
+    {}
+    [[nodiscard]] pair<ComponentFiller*, string> getComponentAndPort1() const { return componentAndPort1; }
+    [[nodiscard]] pair<ComponentFiller*, string> getComponentAndPort2() const { return componentAndPort2; }
+};
+class PortConnexionsManager
+{
+private:
+    vector<PortConnexion> connexions;
+public:
+    void addConnexion(pair<ComponentFiller*, string> componentAndPort1, pair<ComponentFiller*, string> componentAndPort2)
+    {
+        connexions.emplace_back(std::move(componentAndPort1), std::move(componentAndPort2));
+    }
+    vector<pair<ComponentFiller*, string>> getConexionsTo(ComponentFiller* componentFiller, const string& portId)
```

Comment:
linear complexity can be avoided if vector is not used

Reply:
I don't understand how do you suggest we change this?